section/two.py

```diff
@@ -61,48 +66,31 @@ def section_2_3(self):
     def section_2_4(self):
         self.name = "2.4 Ensure CloudTrail trails are integrated with CloudWatch Logs"
         self.scored = True
-        self.passed = True
-
-        if self.CT.describe_trails():
-            resp = self.CT.describe_trails()
-
-            if u'CloudWatchLogsRoleArn' in resp["trailList"][0].keys():
-                CWname = resp["trailList"][0][u'Name']
-                CWstatus = self.CT.get_trail_status(Name=CWname)
-
-                if CWstatus[u'IsLogging']:
-                    LastDelivery = CWstatus[
-                        u'LatestCloudWatchLogsDeliveryTime']
-                    delta = datetime.now(tzlocal()) - LastDelivery
-                    if delta.days > 1:
-                        # logs are considered too old, integration test is
-                        # failed
-                        self.passed = False
-                else:
-                    # currently not Logging
-                    self.passed = False
-            else:  # no CloudWatch integration
-                self.passed = False
-
-        else:
-            # problem with describe_trails
-            self.passed = False
-
+        self.passed = False
+        if self.CT.trailsReport:
+            for trail in self.CT.trailsReport:
+                if 'CloudWatchLogsLogGroupArn' in trail:
+                    trailStatus = self.CT.getTrailStatus(trail["Name"])
+                    date = trailStatus["LatestCloudWatchLogsDeliveryTime"].replace(tzinfo=None)
```

Comment:
i removed the `tzlocal` package, as we're able the remove the time zone using date object